### PR TITLE
Profile page added.

### PR DESCRIPTION
**Pull request by AndriyBilyar**

### CODE REVIEW

File: server/controllers/profile.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+var express = require("express"),
+    router = express.Router(),
+    bodyParser = require('body-parser'),
+    mongoose = require('mongoose'),
+    
+    User = require("../models/user"),
+    Profile = require("../models/profile");
+/*
+router.post("/", function(req, res, next) {
```
Comment (okrasn) [inline server/controllers/profile.js]: why need this commented code?

File: server/controllers/profile.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            'name.first': req.body.name,
+            'name.last': req.body.surname
+        }); */
+
+router.get("/:id", function(req, res, next) {
+
+    Profile
+    .findById(    /*One({"email": newUser.email}) */
+    .exec(function(err, user) { console.log("ahahahahaha");
+        /*if (err) next(err);
```
Comment (okrasn) [inline server/controllers/profile.js]: The same question, why this is commented

File: app/styles/scss/modules/_profile-page.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    .input-group-addon {
+        width: 40px;
+        font-size: 116%;
+        font-weight: bold;
+    }
+
+}
+
+.error {
+    border: 1px solid red;
```
Comment (okrasn) [inline app/styles/scss/modules/_profile-page.scss]: Please use color in hex system not name, in this case just put #FF0000

File: app/modules/profile/view/profileView.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                this.model.save(null, {
+                    success: function(model, response) {
+                        CMS.router.renderHomepage();
+                        CMS.router.navigate("/courses", {trigger: true});
+                    },
+                    error: function(model, response) {
+                    }
+                });
+                this.hideErrors();
+                //CMS.router.navigate("/courses", {trigger: true});
```
Comment (okrasn) [inline app/modules/profile/view/profileView.js]: why this is commented

**Comment by drserg** [comment]
merging all

**Pull request merged**